### ISSUE TITLE
start & stop the mining minigame

### ISSUE THREAD

**Issue by samuelhackwill** [samuelhackwill/le-tiret-du-six#52]
- [x] the mining game will be launched locally by a bookmark calling a function 

#mining  start

the function will need to access the words database and transform the individual words inside of spans in spans of spans
as such : 

from:
``` html
<span> ceci est une ligne de texte </span>
```

to:

``` html
<span> ceci est une 
   <span class="minableWord"> 
      <span class="letter">l</span> 
      <span class="letter">i</span> 
      <span class="letter">g</span> 
      <span class="letter">n</span> 
      <span class="letter">e</span>
   </span>
   de texte
</span>
```

the word game ends either when 

- [x] admin clicks on "end mining" button
- [ ] all the words are collected

**Comment by samuelhackwill** [comment]
what happens if all the words are collected _before_ everybody enters the mining game?

**Comment by samuelhackwill** [comment]
first task done in 3c21b843793722c54c98e3d3f0ec4a4c5e609ce9

**Comment by samuelhackwill** [comment]
second task done in aa855db

**Comment by samuelhackwill** [comment]
last task needs data structure first

**Comment by samuelhackwill** [comment]
last task implemented in 18cc04c but with a rather weak solution : we're using a server side counter wich isn't reset when hot code push happens. It's only reset on server restarts & at the end of the mining game. Will prob work during production but will give confusing results while in dev. Not critical though as the function only triggers a console message server side and is not used to hide words, as this is already done by the harvest letter function.
